config/src/lib.rs

```diff
@@ -125,9 +125,38 @@ pub mod derive {
     }
 }
 
-/// Module handling runtime upgrade logic.
 pub mod runtime_upgrades;
 
+pub mod logger {
+    //! Module containing configuration structures for logger
+
+    use serde::{Deserialize, Serialize};
+
+    const DEFAULT_MAX_LOG_LEVEL: Level = Level::INFO;
+
+    /// Log level for reading from environment and (de)serializing
+    #[allow(clippy::upper_case_acronyms)]
+    #[derive(Debug, Deserialize, Serialize, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+    pub enum Level {
+        /// Error
+        ERROR,
+        /// Warn
+        WARN,
+        /// Info (Default)
+        INFO,
+        /// Debug
+        DEBUG,
+        /// Trace
+        TRACE,
+    }
+
+    impl Default for Level {
+        fn default() -> Self {
+            DEFAULT_MAX_LOG_LEVEL
+        }
+    }
+}
```

Comment:
Just a consideration. Do you think it would be better to move all configurations into the `config` crate, and privatise some of the fields?

Reply:
I did this only because I had to, otherwise dependencies would bleed between crates. Namely, `iroha_data_model` would have to be dependecy of crates with configuration or the other way around and logically they shouldn't be

Reply:
but yes, this can be repeated if dynamic configuration is added for those config fields. Maybe I should comment that somewhere?